Add V2C sensor description (#29762)
home-assistant/home-assistant.github.io: source/_integrations/v2c.markdown

The V2C integration currently exposes the follow sensors:

> - charge_power: current power charging your vehicle

Reviewer: ```suggestion
- charge_power: Current power charging your vehicle.
```

> - charge_energy: energy transferred to your car during the current charging session

Reviewer: ```suggestion
- charge_energy: Energy transferred to your car during the current charging session.
```

> - charge_time: time spent in the current charging session

Reviewer: ```suggestion
- charge_time: Time spent in the current charging session.
```

> - house_power: power consumed by your household (you need to install and configure through the V2C App the CT supplied with your EVSE)

Reviewer: ```suggestion
- house_power: Power consumed by your household. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle support equipment">EVSE</abbr>).
```

> - fv_power: power produced by your photovoltaic system (you need to install and configure through the V2C App the CT supplied with your EVSE)

Reviewer: ```suggestion
- fv_power: Power produced by your photovoltaic system. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle support equipment">EVSE</abbr>).
```

> - house_power: Power consumed by your household. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle support equipment">EVSE</abbr>).
> - fv_power: Power produced by your photovoltaic system. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle support equipment">EVSE</abbr>).

Reviewer: ```suggestion
- house_power: Power consumed by your household. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle supply equipment">EVSE</abbr>).
- fv_power: Power produced by your photovoltaic system. This data is available if you installed and configured the settings in the V2C app for the <abbr title="current transformer">CT</abbr> clamp that was supplied with your <abbr title="electric vehicle supply equipment">EVSE</abbr>).
```
